docs/gen-ai/gen-ai-metrics.md

```diff
@@ -171,6 +175,106 @@ Instrumentations SHOULD document the list of errors they report.
 
 
 
+<!-- markdownlint-restore -->
+<!-- prettier-ignore-end -->
+<!-- END AUTOGENERATED TEXT -->
+<!-- endsemconv -->
+
+## Generative AI Model Server Metrics
+
+The following metric instruments describe Generative AI model servers'
+operational metrics. It includes both functional and performance metrics.
+
+### Metric: `gen_ai.server.latency.request_duration`
+
+This metric is [recommended][MetricRecommended] to report the model server
+latency in terms of time spent per request.
+
+For example, if a GenAI model server reports latency information, it SHOULD be
+used.
+
+If instrumentation cannot obtain this information at a request level and break
+it down into the buckets mentioned below, then it MUST NOT report this metric.
+
+This metric SHOULD be specified with [ExplicitBucketBoundaries] of
+[1.0, 2.5, 5.0, 10.0, 15.0, 20.0, 30.0, 40.0, 50.0, 60.0].
```

Comment:
this one does not seem to be exponential.

Can we reuse the same boundaries as for the client one? If you'd rather change them, let's do it consistently too - I assume there is no reason to expect drastically different time-to-last-token between client and server in case of LLM request.

(I agree lower boundaries are not so helpful for LLM, but there could still be simple operations such as list models or error responses that are in ms range)

Reply:
Makes sense. Made it the same as client buckets.